src/store/index.js

```diff
@@ -58,6 +62,9 @@ export default createStore({
     setDarkMode({ commit }, payload) {
       commit("SET_DARK_MODE", !!payload);
     },
+    setUserData({ commit }, payload) {
```

Comment:
Is this ever used?

Reply:
This refers to a future perspective that will be used to store the current user data.

Reply:
It represents a future aspect to store data of current logged In user. Its not working at this time.